docs/executing-queries.md

```diff
@@ -83,7 +83,8 @@ $psrResponse = $server->processPsrRequest($psrRequest, $psrResponse, $psrBodyStr
 
 /** @var ExecutionResult|ExecutionResult[] $result */
 $result = $server->executePsrRequest($psrRequest);
-$psrResponse = new SomePsr7ResponseImplementation(json_encode($result));
+$jsonResult = json_encode($result, JSON_THROW_ON_ERROR);
```

Comment:
we can even do

JSON_UNESCAPED_UNICODE
 | JSON_UNESCAPED_SLASHES
 | JSON_PRESERVE_ZERO_FRACTION
 | JSON_THROW_ON_ERROR

maybe too much? 😅

Reply:
Maybe in another pull request. I could see adding a global config for the library to control how it uses `json_encode()`.